packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -79,6 +80,13 @@ export interface IPortfolio extends cdk.IResource {
    */
   addProduct(product: IProduct): void;
 
+  /**
+   * Associate Tag Options.
+   * A TagOption is a key-value pair managed in AWS Service Catalog.
+   * It is not an AWS tag, but serves as a template for creating an AWS tag based on the TagOption.
+   */
+  addTagOptions(tagOptions: TagOptions): void;
```

Comment:
1. Let's also allow setting the `TagOptions` directly when creating the `Portfolio`. So, add an optional `readonly tagOptions?: TagOptions` property to `PortfolioProps`.
2. Since you're using "associate" in the description already, how about we call this method `associateTagOptions()`? I'm worried we're overloading the term "add" in this class to mean very many different things.